src/query_ctx.c

```diff
@@ -63,9 +64,18 @@ void QueryCtx_EmitException(void) {
 	}
 }
 
+void QueryCtx_SetGlobalExecCtx(CommandCtx *cmd_ctx) {
```

Comment:
Add function description.
Rename to `QueryCtx_SetGlobalExecutionCtx`

Reply:
done